limitador-server/docs/configuration.md

```diff
@@ -31,6 +31,16 @@ variables:
 - Format: integer.
 
 
+## INFINISPAN_CACHE_NAME
```

Comment:
The amount of environment variables is starting to get out of hand... :)

Yes, this is intended to be run with its own container image, but perhaps it would be good to look into also accepting a configuration file providing fallback settings for missing envs?

Reply:
Yes, we should consider that. There are more config params coming :D